client/src/app/shared/components/list-of-speakers-content/list-of-speakers-content.component.html

```diff
@@ -106,7 +106,8 @@ <h1 class="los-title" *ngIf="!customTitle">
                         </div>
                         <!-- Extra line -->
                         <ng-container
-                            *ngTemplateOutlet="speakerSubtitle; context: { speaker: speaker, showcolor: true }"
+                            [ngTemplateOutlet]="speakerSubtitle"
+                            [ngTemplateOutletContext]="{ speaker: speaker, showcolor: true }"
```

Comment:
Is there a reason why you transform it from a structural directive to an attribute directive?
Nevertheless, it's okay for me and makes the same.

Reply:
Habbit, and it's easier to maintain, since `[ngTemplateOutletContext]` requires only a json that can be slightly easier replaced than in a structural directive. Furthermore, elements can only have one structural directive, thus but as many input and output attributes as required, thus I try to limit the usage to *ngIf, *ngFor or *osPerms